PicaSWB.js

```diff
@@ -104,6 +123,16 @@ function doExport() {
 		} else {
 			titleStatement += item.title.replace(/\s*:\s*/,'$d');
 		}
+		//Sortierzeichen hinzufügen, vgl. https://github.com/UB-Mannheim/zotkat/files/137992/ARTIKEL.pdf
```

Comment:
@zuphilip ja, Nichtsortierzeichen sollte sprachabhängig umgesetzt werden. D.h. aber, dass man in Zotero zwingend das Sprachfeld manuell ergänzt werden muss, falls Item.language beim Download nicht mitkommt.